blocks/single-column-layout-block/layouts/single-column-regular/default.jsx

```diff
@@ -5,25 +5,29 @@ import { Stack } from "@wpmedia/arc-themes-components";
 const LAYOUT_CLASS_NAME = "b-single-column-regular";
 
 const SingleColumnRegular = ({ children }) => {
-	const [navigation, main, footer] = React.Children.toArray(children);
+	const [navigation, body, fullWidth, footer] = React.Children.toArray(children);
```

Comment:
Shouldn't this be in order? 

```suggestion
	const [navigation, fullWidth, body, footer] = React.Children.toArray(children);
```

Reply:
Interesting. I'll see if this affects the order in of the sections appearing in Storybook. Thanks!

Reply:
Oh wow, that did fix the weirdness in the Storybook sections. I should have caught that. TYVM!